src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -551,7 +551,7 @@ def run_end_scenario_sizing(
 
         # We give bounds for stage 2 by allowing one DN sizes larger than what was found in the
         # stage 1 optimization.
-        pc_map = solution._HeatMixin__pipe_topo_pipe_class_map
+        pc_map = solution._pipe_topo_pipe_class_map
```

Comment:
Should you maybe use the function .get_pipe_class_map() instead of ._pipe_topo_pipe_class_map?

Reply:
yup good point